src/js/services/walletService.js

```diff
@@ -220,8 +220,9 @@ angular.module('copayApp.services').factory('walletService', function($log, $tim
       cache.totalBalanceStr = txFormatService.formatAmount(cache.totalBalanceSat) + ' ' + cache.unitName;
       cache.lockedBalanceStr = txFormatService.formatAmount(cache.lockedBalanceSat) + ' ' + cache.unitName;
       cache.availableBalanceStr = txFormatService.formatAmount(cache.availableBalanceSat) + ' ' + cache.unitName;
+      cache.pendingBalanceStr = txFormatService.formatAmount(cache.totalBalanceSat + (cache.pendingAmount === null? 0 : cache.pendingAmount)) + ' ' + cache.unitName;
 
-      if (cache.pendingAmount) {
+      if (cache.pendingAmount !== null && cache.pendingAmount !== 0) {
```

Comment:
I was thinking it would be a good idea to make the logic a little more explicit – reading through the code doesn't make it very obvious. (Maybe we can leave it explicit until we have unit tests for this view?)